src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1128,9 +1125,10 @@ void runtime::cpu::CPU_ExternalFunction::build()
     {
         return;
     }
-
+    // stream writer to dump the debug manifest for the DEX
+    static const string s_debug_dir = "cpu_codegen";
```

Comment:
Why is it still called 'cpu_codegen'?

Reply:
@jbobba wanted to use the same directory `cpu_codegen` for dumping the debug manifest as well.

Reply:
👍